.travis/rpmbuild_test

```diff
@@ -8,21 +8,30 @@ yum install -y openssl-devel libuuid-devel
 
 # switch to an unprivileged user with sudo privileges
 yum install -y sudo
+# We need Git existing here prior to the run
+yum install -y git
 useradd -u 1000 --create-home -s /bin/bash testuser
 echo "Defaults:testuser env_keep=DOCKER_HOST" >>/etc/sudoers
 echo "testuser ALL=(ALL) NOPASSWD: ALL" >>/etc/sudoers
 chown -R testuser .
 
 su testuser -c '
   set -x
-  ./mconfig -V 0.0
+  set -e
+  ./mconfig 
   sudo yum-builddep -y dist/rpm/singularity.spec
+  git fetch --all
+  git config --global user.email "community@sylabs.io"
+  git config --global user.name "Travis Test"
   make -C builddir rpm
   sudo yum install -y $HOME/rpmbuild/RPMS/*/*.rpm
-  BLD=`echo $HOME/rpmbuild/BUILD/singularity-*`
+  BLD=`echo $HOME/rpmbuild/BUILD`
```

Comment:
No, it's important for the singularity-* level to be there.  When people run rpmbuild the rpmbuild/BUILD directory is shared, and packages avoid conflicting with each other by creating a top-level directory of package-version.  This can be fixed by easily added by adding “mkdir %{name}-%{version}” and “cd %{name}-%{version}" to the beginning of the %build step in singularity.spec.in and “cd %{name}-%{version}" to the beginning of "%install", a better solution.